build.sh

```diff
@@ -825,28 +825,28 @@ while :; do
             fi
             ;;
 
-        bindir)
+        bindir|-bindir)
             if [ -n "$2" ]; then
                 __RootBinDir="$2"
                 if [ ! -d $__RootBinDir ]; then
                     mkdir $__RootBinDir
                 fi
-                __RootBinParent=$(dirname $__RootBinDir)
+                __RootBinParent=$(dirname $__RootBinDir|-__RootBinParent=$(dirname $__RootBinDir)
```

Comment:
This should not be changed - I guess it is a search and replacement artifact, right?